code/modules/admin/verbs/map_template_loadverb.dm

```diff
@@ -26,6 +26,21 @@
 			to_chat(src, "Failed to place map", confidential = TRUE)
 	images -= preview
 
+/client/proc/ship_template_load()
+	set category = "Debug"
+	set name = "Load Template As Ship"
+
+	var/datum/map_template/template
+
+	var/map = input(src, "Choose a Map Template to spawn as a ship.","Place Map Template") as null|anything in sortList(SSmapping.map_templates)
+	if(!map)
+		return
+	template = SSmapping.map_templates[map]
```

Comment:
```suggestion
	var/datum/map_template/template = SSmapping.map_templates[input(src, "Choose a Map Template to spawn as a ship.","Place Map Template") as null|anything in sortList(SSmapping.map_templates)]
```
this one's kind of up to choice, just putting it here if you want it to be one line, don't feel forced to do this